### ISSUE TITLE
Implement impossible to schedule in UI

### ISSUE THREAD

**Issue by tijlleenders** [tijlleenders/ZinZen#896]
Show impossible only when present.
Implement similar to MyGoals/Inbox UI.

**Comment by tijlleenders** [comment]
Blocked by https://github.com/tijlleenders/ZinZen/issues/882

**Comment by Tushar-4781** [comment]
Duplicate of #968 
Resolved by #969
